oe_theme.theme

```diff
@@ -1102,6 +1152,13 @@ function oe_theme_preprocess_block__system_branding_block(&$variables) {
 
   $route_options = ['absolute' => TRUE];
   $variables['site_logo_href'] = Url::fromRoute('<front>', [], $route_options)->toString();
+
+  // Ensure the site logo falls back to english version if the language is
+  // not EU language.
+  $language_code = \Drupal::languageManager()->getCurrentLanguage()->getId();
```

Comment:
We also have the logo in the footer when the European Union theme flavour is used:

![Screen Shot 2021-01-12 at 09 19 51](https://user-images.githubusercontent.com/5848933/104287914-6c709d00-54b7-11eb-8c3f-3b7dd6fa64b3.png)